### PR TITLE
Update index.mdx

### PR DESCRIPTION
**Pull request by ftouserkani-edb**
BigAnimal free trial users keep going around in circles attempting to log in to BigAnimal portal.
This PR tends to clear that confusion and point the free trial users to the correct sute when attempting to login into the portal.   

Grammatical updates are left to the discretion of the documentation folks.

- [ ] This PR adds new content
- [x] This PR changes existing content
- [ ] This PR removes existing content
- [ ] This PR is for a release, please add this tag:

### CODE REVIEW

File: product_docs/docs/biganimal/release/free_trial/detail/create_an_account/index.mdx [hunk 1]
```diff
@@ -4,17 +4,18 @@ description: "Creating an EDB account and signing into the BigAnimal free trial
 navTitle: "Create an account" 
 ---
 
-Registering an account on [enterprisedb.com](https://www.enterprisedb.com/) gets you access to the BigAnimal trial, as well as EDB's training and software downloads. 
+You will need to register with EDB in order to access BigAnimal free trial, as well as EDB's training and software downloads.
```
Comment (ebgitelman) [inline product_docs/docs/biganimal/release/free_trial/detail/create_an_account/index.mdx:7]: You need to register with EDB to access the BigAnimal free trial and EDB's training and software downloads.

(no future tense, "in order to" is wordy)

File: product_docs/docs/biganimal/release/free_trial/detail/create_an_account/index.mdx [hunk 2]
```diff
@@ -4,17 +4,18 @@ description: "Creating an EDB account and signing into the BigAnimal free trial
...
 Accounts are free, and registration takes less than a minute.
 
 1. Navigate to <https://biganimal.com>.
-2. Select the **Try it for free** button.
-3. Fill in the form, and agree to the terms and conditions.
+2. Select the **Test drive for free** button.
+3. Fill in the form, and agree to the **Terms and Conditions** and the **Privacy Policy**.
```
Comment (ebgitelman) [inline product_docs/docs/biganimal/release/free_trial/detail/create_an_account/index.mdx:12]: Select **Test drive for free**.
Fill in the form, and agree to the terms and conditions and privacy policy.

File: product_docs/docs/biganimal/release/free_trial/detail/create_an_account/index.mdx [hunk 3]
```diff
@@ -4,17 +4,18 @@ description: "Creating an EDB account and signing into the BigAnimal free trial
...
 4. Select **Submit**.
 5. Check your email. You should receive an email from EDB containing an activation link.
-6. Select the activation link and enter a password for your EDB account.
+6. Click the activation link and enter a password for your EDB account.
```
Comment (ebgitelman) [inline product_docs/docs/biganimal/release/free_trial/detail/create_an_account/index.mdx:15]: Select was correct. Discard this edit.

File: product_docs/docs/biganimal/release/free_trial/detail/create_an_account/index.mdx [hunk 4]
```diff
@@ -4,17 +4,18 @@ description: "Creating an EDB account and signing into the BigAnimal free trial
...
 
 You're then redirected to the BigAnimal portal.
 
 !!! Note "Already have an EDB account?"
-    If you've already signed up for an EDB account, you can navigate to <https://portal.biganimal.com/> to log in.
+    If you've already signed up for an EDB account, you can navigate to <https://freetrial.biganimal.com/> to log in.
+    Please continue to use this portal in order to access BigAnimal for the duration of your free trial.
```
Comment (ebgitelman) [inline product_docs/docs/biganimal/release/free_trial/detail/create_an_account/index.mdx:21]: If you already signed up for an EDB account, navigate to <https://freetrial.biganimal.com/>  to log in. Continue to use this portal to access BigAnimal for the rest of your free trial.

File: product_docs/docs/biganimal/release/free_trial/detail/create_an_account/index.mdx [hunk 5]
```diff
@@ -4,17 +4,18 @@ description: "Creating an EDB account and signing into the BigAnimal free trial
...
+You will need to register with EDB in order to access BigAnimal free trial, as well as EDB's training and software downloads.
 Accounts are free, and registration takes less than a minute.
 
 1. Navigate to <https://biganimal.com>.
-2. Select the **Try it for free** button.
-3. Fill in the form, and agree to the terms and conditions.
+2. Select the **Test drive for free** button.
+3. Fill in the form, and agree to the **Terms and Conditions** and the **Privacy Policy**.
 4. Select **Submit**.
 5. Check your email. You should receive an email from EDB containing an activation link.
```
Comment (ebgitelman) [inline product_docs/docs/biganimal/release/free_trial/detail/create_an_account/index.mdx:14]: Check your email for an email from EDB that contains an activation link.

**Review by ebgitelman — CHANGES_REQUESTED** [review]

**Review by drothery-edb — APPROVED** [review]

**Pull request merged**